import-export-cli/cmd/mi/add/addEnv.go

```diff
@@ -0,0 +1,75 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package add
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/impl"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+var envToBeAdded string         // Name of the environment to be added
+var miManagementEndpoint string // mi management endpoint of the environment to be added
+
+// AddEnv command related Info
+const AddEnvCmdLiteral = "env [environment]"
```

Comment:
Since we are passing the MI management endpoint as an argument, let's update this as follows,
```suggestion
const AddEnvCmdLiteral = "env [environment] [mi_management_url]"
```